src/components/Chat.jsx

```diff
@@ -2,9 +2,15 @@ import React from "react";
 import moment from "moment";
 
 function Chat({ message }) {
+  console.log('CHAT.JSX - MESSAGE : ', message)
+  if (typeof message.message === 'object') {
```

Comment:
is this necessary? seems like the message is g=being processed on the backend to be in the message property